src/pages/public_registration/action_accordions/FindRegistration.tsx

```diff
@@ -0,0 +1,106 @@
+import SearchIcon from '@mui/icons-material/Search';
+import { Autocomplete, Box, IconButton, TextField, Typography } from '@mui/material';
+import { useQuery } from '@tanstack/react-query';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { fetchResults, FetchResultsParams } from '../../../api/searchApi';
+import { ErrorBoundary } from '../../../components/ErrorBoundary';
+import { RegistrationListItemContent } from '../../../components/RegistrationList';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { Registration } from '../../../types/registration.types';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+
+function isDoi(query: string) {
+  return query.includes('https://doi.org/');
```

Comment:
Det kommer til å dukke opp en ny oppgave for å støtte det folk flest kaller en dei, nemlig det som er _etter_ `https://doi.org/`.
Om du er interessert kan du se i `LinkRegistration.tsx` hvordan vi har gjort det der, hvor vi sjekker for selve DOI i stedet for prefiks.
Øverste [journal article i ntnu open](https://ntnuopen.ntnu.no/ntnu-xmlui/handle/11250/2500050) per nå har doi: `http://dx.doi.org/10.3389/fpls.2017.00233`. Antar vi tar vare på den prefiksen i importer, så veldig mange faktiske DOI-URIer vil også feile sjekken her nå faktisk 😅 
Men dette kan godt rettes senere. Tipper 90% vil søke på tittel eller bidragsyter her

Reply:
dx.doi.org vil bli konvertert til doi.org mener jeg. 
https://github.com/BIBSYSDEV/nva-commons/blob/main/doi/src/test/java/nva/commons/doi/DoiConverterTest.java

Er ikke så lett lese av testen umiddelbart men det er en test som sjekker at dx.doi.org/10.3389/fpls.2017.00233 ville blitt konvertert til https://doi.orgd/10.3389/fpls.2017.00233

Reply:
når jeg tenker meg om vil det sikkert bli et meningsfylt søketreff likevel.
`isDoi(10.3389/fpls.2017.00233)` -> `false`, men da vil vi søke på `10.3389/fpls.2017.00233` som fritekst med `query`, og det burde jo gi samme svar egentlig